src/Band/StringInstruments/Guitar.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace Band\StringInstruments;
+
+/**
+ * Class Guitar
+ */
+class Guitar extends StringInstrument
+{
+    /**
+     * @return int
+     */
+    function getNumberOfStrings()
+    {
+        return 6;
```

Comment:
Guitar can have 7 chords and 12 chords as well. So either you convert this class to abstract or please get your variant.